core-services/image-mirroring/knative/mapping_knative_v0_23_quay

```diff
@@ -0,0 +1,13 @@
+registry.ci.openshift.org/openshift/knative-v0.23:knative-eventing-test-event-flaker quay.io/openshift-knative/event-flaker:v0.23
```

Comment:
@cardil  from release 0.23 on, we just do `0.23` version. There will be just one branch. All upstream fixes will be cherry picked to the `release-vX.Y` branches.

The only thing that really uses the quay.io images is the upgrade test, on the serverless-operator, only.  Please check if that schema makes sense for you (starting w/ Serverless 1.17 this will be a thing)